onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_execution_provider.cc

```diff
@@ -121,16 +123,33 @@ NnapiExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph_view
 
   std::unordered_set<std::string> node_outputs_in_current_group{};
 
+  // Add QDQ selector and obtain qdq node groups in the graph before checking if a node is supported
+  params.qdq_support_helper->GetQDQNodeGroups(graph_viewer);
```

Comment:
>GetQDQNodeGroups [](http://example.com/codeflow?start=29&length=16)

could this be done in the QDQSupportHelper constructor? if so, maybe params can stay const

Reply:
Moved this to constructor and changed params back to const.